### PR TITLE
feat: new feature choice for Style Resources

### PR DESCRIPTION
**Pull request by carlos-avila**

### CODE REVIEW

File: template/_package.json [hunk 1]
```diff
@@ -119,6 +119,9 @@
     <%_ if (ui === 'tachyons') { _%>
     "tachyons": "^4.11.1",
     <%_ } _%>
+    <%_ if (styleResources) { _%>
+    "@nuxtjs/style-resources": "^1.0.0",
```
Comment (clarkdo) [inline template/_package.json]: Can we put the package as alphabetical order ?

Comment (carlos-avila) [reply template/_package.json]: Sorted the packages in alpha order and moved the conditional for tachyons up as part of the rest of the of 'ui ===' group.

**Comment by Atinux** [comment]
Thank you very much @carlos-avila

Actually, after discussion with the team, it's better to not include such module at the beginning but instead having an easier way to add a module from nuxt directly (ie: `npx nuxt add style-resources`)

When bootstraping a Nuxt app, few people actually now these module and can confuse them 😬 

We really do appreciate your help 💚

**Comment by carlos-avila** [comment]
That sounds very interesting. I'd like to help with something like that. Let me know where I should start.

**Pull request closed without merging**